Service/Request/RequestFactory.php

```diff
@@ -71,7 +71,7 @@ public function __construct(
         RequestVisitorRegistryInterface $requestVisitorRegistry,
         UriFactory $uriFactory,
         MessageFactory $messageFactory,
-        bool $strictModeEnabled
+        bool $strictModeEnabled = false
```

Comment:
`false` is set here, too:

https://github.com/auto1-oss/service-api-client-bundle/blob/99584b15312936e2aa506742728ecc6aec0fe850/DependencyInjection/Configuration.php#L37